### ISSUE TITLE
Design customizations

### ISSUE THREAD

**Issue by mishal** [jangouts/jangouts#191]
We are evaluating usage of jangouts for rtc communication, but we need to customize the ui graphics to match our designs. We found some ui glitches, which will be submitted as separate issues.

Which branch is the "main"? It seems that there is an activity in "angular-2" branch... There is also "new-look" branch.

Thanks

**Comment by mishal** [comment]
This relates to https://github.com/openSUSE/mentoring/issues/80

**Comment by code-master5** [comment]
@mishal Since latest commits are done on `master` branch, I guess it's the "main" branch. There are also some new features added to master branch which are not in `angular-2` branch.

Btw, what are the UI glitches you mentioned?

**Comment by imobachgs** [comment]
Yes, for the time being, `master` is the right branch. The angular-2 branch is not completed yet.

**Comment by mishal** [comment]
Thanks. I will use the master branch.

Ui glitches (in short): 

 * UI "jumping" in firefox
 * autofocus on speaking person cannot be enabled again

I will prepare more detailed info.

I can help with the ui, but I not familiar yet with the codebase.

**Comment by ancorgs** [comment]
The jumping is caused by this component https://github.com/ManifestWebDesign/angular-gridster Maybe the fix is as easy as updating.

Clicking again in the "manually focused" participant re-enables autofocus. Is not working?

**Comment by mishal** [comment]
* The gridster requirement is bower.json is defined as ~0.13.5. The latest version should be covered (tilde)...
* Clicking again manually focused is not very intuitive, I did not know that it works like this...

**Comment by ancorgs** [comment]
@mishal BTW, we are conducting a survey about a redesign of the Jangouts UI. It would be very appreciated if you take it. Don't forget to mention theming support at some point. https://goo.gl/forms/JJj6WCiYyhUpeURS2
